index.html

```diff
@@ -14,18 +14,62 @@
   <link rel="stylesheet" href="https://cdn.rawgit.com/konpa/devicon/df6431e323547add1b4cf45992913f15286456d3/devicon.min.css">
   <link href="https://fonts.googleapis.com/css?family=Gloria+Hallelujah|Varela+Round" rel="stylesheet">
   <link rel="stylesheet" type="text/css" href="css/style.css" />
-  <title>Nonprofit Product Landing Page Design - Lunatic Apps</title>
+  <!-- Font -->
+  <link href="https://fonts.googleapis.com/css?family=Muli:400,700" rel="stylesheet">
+  <!-- Animate CSS -->
+  <link rel="stylesheet" href="css/animate.css">
 
 </head>
   <body>
-
+    <!-- START SECTION FEATURE#13, BUILT BY MARCIN WICHA-->
+    <div class="row section-features">
```

Comment:
Use lowercase comment when working in small sections. I need one thing to you use start and end comments on the specific <div> sections. Like this:
```
<!--- start feat: why choose lunatic--->
<div class="row section-features">
      <!-- start title -->
      <div class="col-12">
        <h2 class="feature-title">Why Choose Lunatic</h2>
      </div>
      <!-- end title -->
</div>
<!--- end feat: why choose lunatic--->
```